pages.zh/common/touch.md

```diff
@@ -0,0 +1,20 @@
+# touch
+
+> 创建文件并设置访问/修改时间。
+> 更多信息：<https://manned.org/man/freebsd-13.1/touch>。
```

Comment:
```suggestion
> 更多信息：<https://manned.org/man/freebsd-13.1/touch>.
```

Reply:
Thank you for your suggestion. I have made the changes according to your recommendation.